src/cargo.rs

```diff
@@ -0,0 +1,23 @@
+use std::process::Command;
+
+pub fn update_lockfile(repository_path: &str) -> bool {
+    let manifest_path = format!("{}/Cargo.toml", repository_path);
+    Command::new("cargo")
+        .arg("fetch")
+        .arg("--manifest-path")
+        .arg(manifest_path)
+        .status()
+        .map(|s| s.success())
+        .unwrap_or(false)
+}
+
+pub fn package(repository_path: &str) -> bool {
+    let manifest_path = format!("{}/Cargo.toml", repository_path);
+    Command::new("cargo")
+        .arg("package")
+        .arg("--manifest-path")
+        .arg(manifest_path)
+        .status()
+        .map(|s| s.success())
+        .unwrap_or(false)
```

Comment:
That's why we're returning false. Not really proper error handling (yet), but we still abort in main if we get `false` for now